python/py-graph-tool/Portfile

```diff
@@ -42,21 +42,12 @@ if {${os.major} <= 12 && ${os.platform} eq "darwin"} {
         compiler.cxx_standard   2017
 
         variant openmp description "Enable OpenMP" {
-            # make sure libomp is installed at runtime, even if the compiler gets uninstalled
-            depends_lib-append    lib:${prefix}/lib/libomp/libomp:libomp
+            # TODO: verify OpenMP version required
+            compiler.openmp_version 2.5
             configure.args-append --enable-openmp
         }
 
-        variant clang60 requires openmp conflicts clang70 description "Use clang-6.0 and enable OpenMP" {
-            configure.compiler  macports-clang-6.0
-        }
-        variant clang70 requires openmp conflicts clang60 description "Use clang-7.0 and enable OpenMP" {
-            configure.compiler  macports-clang-7.0
-        }
         default_variants +openmp
-        if {![variant_isset clang60]} {
-            default_variants-append +clang70
-        }
```

Comment:
My earlier comment from #6915:

> My understanding is that these variants were originally added to ensure a compiler supporting OpenMP could be selected. Given the availability of the `compiler.openmp_version` option, which automatically selects an OpenMP-capable compiler, I believe these variants are no longer necessary.
> 
> I have not determined the exact OpenMP version required, but if this is known it can be specified instead of `2.5`. Currently the `compiler.cxx_standard 2017` option is more likely to influence the minimum version of a compiler used (e.g. clang 5.0/GCC 7 or later).